web/views/run_view.ex

```diff
@@ -23,4 +22,8 @@ defmodule Gatherto.RunView do
     "#{min_distance} to #{max_distance} #{units}s."
   end
 
+  def google_maps_api do
+    api_key = System.get_env("GOOGLE_MAPS_API_KEY")
```

Comment:
Do we need to update the documentation to tell devs how to generate a maps API key or is it the same as the other key?